handbook/engineering/README.md

```diff
@@ -29,6 +28,23 @@ The metrics are:
 
 Each week these are tracked and shared in the weekly KPI sheet by Luke Heath.
 
+### Write a feature guide 
+We write [guides](https://fleetdm.com/guides) for all new features. Feature guides are published before the feature is released so
+that our users understand how the feature is intended to work. A guide is a type of
+article, so the process for writing a guide and article is the same.
```

Comment:
I realize that this will format all on the same line when it's rendered and the separate lines make it easier to digest instead of having to scroll and read. I think the line breaks through me off (re my suggestion on lines 43-46). Feel free to disregard if intentional.

Reply:
Not intentional, my IDE is inserting them and driving me crazy. Thanks for catching.